common/src/autogluon/common/utils/log_utils.py

```diff
@@ -193,3 +193,32 @@ def reset_logger_for_remote_call(verbosity: int):
     # Limiting the verbosity of the BaggedEnsembleModel logger to 10 to avoid
     # duplicated messages about fitting.
     set_logger_verbosity(verbosity=min(verbosity, 1), logger=bem_logger)
+
+
+def warn_if_mlflow_autologging_is_enabled(logger: Optional[logging.Logger] = None):
```

Comment:
Any ideas for how/whether we can test this method in our unittests without installing mlflow?

Reply:
rather test the good path?

(pseudocode)

```
def test():
    try:
         import mlflow
         pytest.fail("mlflow not expected to be installed in test env")
    except:
         assert warn_if_mlflow_autologging_is_enabled()  does not log
```